src/modules/differential_drive/DifferentialDriveControl/DifferentialDriveControl.cpp

```diff
@@ -89,6 +90,16 @@ void DifferentialDriveControl::control(float dt)
 		}
 	}
 
+	if (_vehicle_status_sub.updated()) {
```

Comment:
I would have put that outside since the controller should just be called when it needs to be run and not otherwise.

Reply:
I put this in 6e4d80776c1b48e72f4cc4c634b14c3bfc91e829